### DISCUSSION TITLE
[Suggestion] Add 2004 Graphics / Original Models / Old UI

### DISCUSSION THREAD

**Discussion by tylerguitar75** [runelite/runelite#15233] [Ideas]
With the popularity of the HD RuneLite Plugin, it would be amazing if someone created a true oldschool graphics plugin.

The 2004 models and textures are the most nostalgic for many players, and it would be great if someone could add them to the client.

Here is a gif showing some of the old models (just monsters - would need environment too):
https://imgur.com/gallery/cWsPt

Obviously if this takes off, it would be good to open dialog with Jagex.

Examples:
![image](https://user-images.githubusercontent.com/9688189/176255684-9d3b9d7f-27cb-468a-b9f2-af204fb7aeff.png)
![image](https://user-images.githubusercontent.com/9688189/176255774-d48e3c3a-1cde-46e7-8257-e7db2f161337.png)

**Comment by Mark7625** [comment]
I don't think we can edit models, if we can be amazing to have

**Reply by tylerguitar75** [reply]
At the bare minimum, we could get the old textures back in. For example - the original gray walls of Falador.
